src/Module/BaseApi.php

```diff
@@ -60,17 +55,6 @@ class BaseApi extends BaseModule
 
 	public static function init(array $parameters = [])
```

Comment:
You can remove the whole method if there's nothing left in it.

Reply:
I was unsure if this was needed or used at any place. Also I saw other `init` functions that are empty as well. When you think that it is safe to remove it, I will do so.

Reply:
Since `BaseApi` extends `BaseModule`, the `init()` method will always exist in classes extending `BaseApi`. So you can remove this empty method in any class that is NOT `BaseModule`.